src/pages/HomePage.js

```diff
@@ -34,55 +34,61 @@ const HomePage = () => {
     return (
         <div>
             {/* PIC HEADER */}
-            <div className='bg-homePagePic bg-cover h-100 bg-center '>
-                <div className='flex justify-center p-4 font-HomePageHeading font-semibold text-black '>
-                    <h1 className='text-6xl'>
+            <div className='bg-homePagePic bg-cover h-[100vh] bg-center'>
```

Comment:
I would remove the `bg-homePagePic` image, I think Austin added that just to have something initially lol. The bank will probably not like it, I think it could be removed entirely. Maybe replace it with something if it looks too bare

Reply:
LOL, I was wondering about that image.   We'll probably need a little something back there because it could seem too bare as you stated.   If you or anyone has any ideas or suggestions, let me know.  Whatever works!  Totally flexible about it.